python/src/aac/plugins/validators/reference_type/_reference_type.py

```diff
@@ -0,0 +1,70 @@
+from aac.lang.definitions.definition import Definition
+from aac.lang.definitions.structure import get_substructures_by_type
+from aac.lang.language_context import LanguageContext
+from aac.plugins.validators import ValidatorFindings, ValidatorResult
+
+
+PLUGIN_NAME = "Reference is type"
+
+
+def validate_reference_types(
+    definition_under_test: Definition,
+    target_schema_definition: Definition,
+    language_context: LanguageContext,
+    *validation_args,
+    **validation_kw_args,
```

Comment:
Python usually uses `**kwargs` to represent keyword arguments, I think it would be good to keep in line with that style.

```suggestion
    **validation_kwargs,
```